Failed building server

After downloading the current master, I tried the following and ran into the same problem on both Windows and Ubuntu.  I guess I'm doing something wrong, but I don't know what....

```code
demo@ubuntu:~/java-webauthn-server$ ./gradlew :webauthn-server-core:jar
Downloading https://services.gradle.org/distributions/gradle-6.8-bin.zip
..........10%..........20%..........30%...........40%..........50%..........60%..........70%...........80%..........90%..........100%

Welcome to Gradle 6.8!

Here are the highlights of this release:
 - Faster Kotlin DSL script compilation
 - Vendor selection for Java toolchains
 - Convenient execution of tasks in composite builds
 - Consistent dependency resolution

For more details see https://docs.gradle.org/6.8/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :yubico-util
WARNING: Skipping source set 'main' as the Scala version could not be detected

> Configure project :webauthn-server-core-minimal
WARNING: Skipping source set 'main' as the Scala version could not be detected

> Configure project :webauthn-server-attestation
WARNING: Skipping source set 'main' as the Scala version could not be detected
WARNING: Skipping source set 'test' as the Scala version could not be detected

FAILURE: Build failed with an exception.

* Where:
Build file '/home/demo/java-webauthn-server/build.gradle' line: 124

* What went wrong:
A problem occurred evaluating project ':webauthn-server-core-minimal'.
> Failed to get git commit ID

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/build.gradle
+++ b/build.gradle
@@ -121,9 +121,13 @@ String getGitCommit() {
   def proc = "git rev-parse HEAD".execute(null, projectDir)
   proc.waitFor()
   if (proc.exitValue() != 0) {
-    throw new RuntimeException("Failed to get git commit ID");
+    return null
   }
   return proc.text.trim()
+}
+
+String getGitCommitOrUnknown() {
+  return getGitCommit() ?: 'UNKNOWN'
 }
 
 subprojects { project ->
@@ -211,6 +215,10 @@ subprojects { project ->
 
     apply plugin: 'maven-publish'
     apply plugin: 'signing'
+
+    if (getGitCommit() == null) {
+      throw new RuntimeException("Failed to get git commit ID");
+    }
 
     publishing {
       publications {

--- a/test-dependent-projects/java-dep-webauthn-server-attestation/src/test/java/com/yubico/webauthn/attestation/ManifestInfoTest.java
+++ b/test-dependent-projects/java-dep-webauthn-server-attestation/src/test/java/com/yubico/webauthn/attestation/ManifestInfoTest.java
@@ -36,6 +36,7 @@ public class ManifestInfoTest {
 
   @Test
   public void customImplementationPropertiesAreSet() throws IOException {
-    assertTrue(lookup("Git-Commit").matches("^[a-f0-9]{40}$"));
+    assertTrue(
+        lookup("Git-Commit").matches("^[a-f0-9]{40}$") || lookup("Git-Commit").equals("UNKNOWN"));
   }
 }

--- a/test-dependent-projects/java-dep-webauthn-server-core/src/test/java/com/yubico/webauthn/meta/ManifestInfoTest.java
+++ b/test-dependent-projects/java-dep-webauthn-server-core/src/test/java/com/yubico/webauthn/meta/ManifestInfoTest.java
@@ -54,6 +54,7 @@ public class ManifestInfoTest {
 
   @Test
   public void customImplementationPropertiesAreSet() throws IOException {
-    assertTrue(lookup("Git-Commit").matches("^[a-f0-9]{40}$"));
+    assertTrue(
+        lookup("Git-Commit").matches("^[a-f0-9]{40}$") || lookup("Git-Commit").equals("UNKNOWN"));
   }
 }

--- a/test-dependent-projects/java-dep-webauthn-server-core/src/test/java/com/yubico/webauthn/meta/VersionInfoTest.java
+++ b/test-dependent-projects/java-dep-webauthn-server-core/src/test/java/com/yubico/webauthn/meta/VersionInfoTest.java
@@ -28,14 +28,17 @@ public class VersionInfoTest {
     final Implementation impl = versionInfo.getImplementation();
     assertTrue(impl.getSourceCodeUrl().toExternalForm().startsWith("https://"));
     assertTrue(impl.getVersion().matches("^\\d+\\.\\d+\\.\\d+(-.*)?"));
-    assertTrue(impl.getGitCommit().matches("^[a-f0-9]{40}$"));
+    assertTrue(
+        impl.getGitCommit().matches("^[a-f0-9]{40}$") || impl.getGitCommit().equals("UNKNOWN"));
   }
 
   @Test
-  public void majorVersionIsAtLeast1() {
+  public void majorVersionIsUnknownOrAtLeast1() {
     final String version = versionInfo.getImplementation().getVersion();
-    String[] splits = version.split("\\.");
-    final int majorVersion = Integer.parseInt(splits[0]);
-    assertTrue(majorVersion >= 1);
+    if (!"0.1.0-SNAPSHOT".equals(version)) {
+      String[] splits = version.split("\\.");
+      final int majorVersion = Integer.parseInt(splits[0]);
+      assertTrue(majorVersion >= 1);
+    }
   }
 }

--- a/webauthn-server-attestation/build.gradle
+++ b/webauthn-server-attestation/build.gradle
@@ -58,7 +58,7 @@ jar {
       'Implementation-Title': project.description,
       'Implementation-Version': project.version,
       'Implementation-Vendor': 'Yubico',
-      'Git-Commit': getGitCommit(),
+      'Git-Commit': getGitCommitOrUnknown(),
     ])
   }
 }

--- a/webauthn-server-core-bundle/build.gradle
+++ b/webauthn-server-core-bundle/build.gradle
@@ -27,7 +27,7 @@ jar {
       'Implementation-Version': project.version,
       'Implementation-Vendor': 'Yubico',
       'Implementation-Source-Url': 'https://github.com/Yubico/java-webauthn-server',
-      'Git-Commit': getGitCommit(),
+      'Git-Commit': getGitCommitOrUnknown(),
     ])
   }
 }

--- a/webauthn-server-core/build.gradle
+++ b/webauthn-server-core/build.gradle
@@ -67,7 +67,7 @@ jar {
       'Implementation-Version': project.version,
       'Implementation-Vendor': 'Yubico',
       'Implementation-Source-Url': 'https://github.com/Yubico/java-webauthn-server',
-      'Git-Commit': getGitCommit(),
+      'Git-Commit': getGitCommitOrUnknown(),
     ])
   }
 }

